apps/webapp/app/send/send-form.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import React, { useState } from 'react';
+import { InputBlock, InputToken, ResponsiveImage } from '../../shared';
+import { Button, Switch } from 'ui';
+import { Asset } from './types';
+import { assets } from './constants';
+
+export const SendForm = () => {
+  const [asset, setAsset] = useState<Asset>(assets[0]!);
+  const [recepient, setRecepient] = useState('');
+  const [memo, setMemo] = useState('');
+  const [amount, setAmount] = useState('');
+  const [hidden, setHidden] = useState(false);
+
+  return (
+    <form
+      className='flex flex-col gap-2'
+      onSubmit={e => {
+        e.preventDefault();
+      }}
+    >
+      <InputBlock
+        label='Recepient'
+        placeholder='Enter the address'
+        className='mb-1'
+        value={recepient}
+        onChange={e => setRecepient(e.target.value)}
+      />
```

Comment:
We need some input validation that this is a valid penumbra address. Maybe there is some regex we can use to verify it.

Reply:
Yes, that would be nice to have. That's probably why zustand may be easier to use for this as it makes testing easier as well.